manual/qmcpack_manual.tex

```diff
@@ -169,6 +169,8 @@ \chapter{Trial wavefunction specification}
 \input{opt}
 \input{dmc}
 \input{reptation}
+\input{afqmc}
```

Comment:
The AFQMC section in the manual is probably better in its own chapter.    It would keep the current 'Methods' chapter from getting too large, and it doesn't share very much in terms of input file parameters.

Reply:
I will move it to its own section. The manual still needs work, which I will work on later this week. The PySCF section is very rough right now and needs many more things. I also need to upload the python scripts that I use to calculate the integrals, e.g qmctools. I'm mainly interested right now in fixing issues with the AFQMC code that appear in other architectures, since I only compile with gnu on a Xeon machine.

Reply:
Moved to its own section.